src/main/java/core/basesyntax/csv/reader/ReadFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.csv.reader;
```

Comment:
I doubt about package, I think its better to store them with impl in service package. The same for writeToFile class and impl